lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -137,5 +137,26 @@ svg.nx-radio-checkbox__control {
         stroke: var(--nx-swatch-blue-40);
       }
     }
+
+    &:disabled{
+      + .nx-radio-checkbox__control {
+        &.nx-checkbox__box {
```

Comment:
This file is actually only for the styles that are common between `NxRadio` and `NxCheckbox`.  Styles that are specific to each should be placed in the `NxCheckbox.scss` and `NxRadio.scss` files.

Reply:
> This file is actually only for the styles that are common between `NxRadio` and `NxCheckbox`. Styles that are specific to each should be placed in the `NxCheckbox.scss` and `NxRadio.scss` files.

Split out the styles added on `_nx-radio-checkbox.scss` and placed the accordingly into `NxCheckbox.scss` and `NxRadio.scss.

[a429dc9](https://github.com/sonatype/sonatype-react-shared-components/commit/a429dc9e51134c98d0bff427f35ff5056e85151d)